drivers/d3d12/rendering_device_driver_d3d12.cpp

```diff
@@ -3772,7 +3784,7 @@ RDD::UniformSetID RenderingDeviceDriverD3D12::uniform_set_create(VectorView<Boun
 
 	{
 		uniform_set_info->resource_states.reserve(resource_states.size());
-		uint32_t i = 0;
+		[[maybe_unused]] uint32_t i = 0;
```

Comment:
It seems to be really unused, probably better to remove or comment it out (along with `i++` later).